dockstore-common/src/main/java/io/dockstore/common/yaml/Service12.java

```diff
@@ -51,4 +51,8 @@ public String getPrimaryDescriptorPath() {
     public String getReadMePath() {
         throw new UnsupportedOperationException();
     }
+
+    public String getTopic() {
+        throw new UnsupportedOperationException();
```

Comment:
An aside, is the topic not supported for Services because we didn't add the support when we first implemented topics?

Reply:
I believe the support is there in the backend, but not exposed in the front-end for services. I followed the same pattern as the recently added `readMePath` property where it's not supported for services either